lib/symbol-store.coffee

```diff
@@ -9,34 +9,28 @@ class Symbol
   type: null
 
   constructor: (@text) ->
-    @metadataByPath = {}
+    @metadataByPath = new Map
```

Comment:
@nathansobo I'm using a `Map` in the `SymbolStore` so I can iterate it. Seems `WeakMap` doesnt allow for iteration.